src/app/app.config.ts

```diff
@@ -16,6 +16,7 @@
 
 import { ApplicationConfig, provideZoneChangeDetection } from '@angular/core';
 import { initializeApp, provideFirebaseApp, getApp } from '@angular/fire/app';
+// TODO: Recommended to use together with Vertex AI
```

Comment:
In addition to adding a comment line here, wdyt about also _commenting out_ this import statement [1] to make it clear that it's not needed until you decide to start using Vertex AI?

[1]
```
import {
  ReCaptchaEnterpriseProvider,
  initializeAppCheck,
  provideAppCheck,
} from '@angular/fire/app-check';
```